app/_kong_plugins/session/index.md

```diff
@@ -31,6 +31,97 @@ categories:
 
 search_aliases:
   - sessions
+
+related_resources:
+  - text: Authenticate Consumers with the Key Auth and Sessions plugins
+    url: /how-to/authenticate-consumers-with-session-and-key-auth/
 ---
 
-## Overview
+The Session plugin can be used to manage browser sessions for APIs proxied
+through the {{site.base_gateway}}. It provides configuration and management for
+session data storage, encryption, renewal, expiry, and sending browser cookies.
+It is built using
+[lua-resty-session](https://github.com/bungle/lua-resty-session).
+
+## How it works
+
+The Session plugin can be configured globally or with an entity (for example, a [Gateway Service](/gateway/entities/service/) or a [Route](/gateway/entities/route/))
+and is always used in conjunction with another [{{site.base_gateway}} authentication plugin](/plugins/?category=authentication). This
+plugin is intended to work similarly to the [multiple authentication setup](/gateway/authentication/#using-multiple-authentication-methods).
+
+When the Session plugin is enabled in conjunction with an authentication plugin,
+it runs before credential verification. If no session is found, then the
+authentication plugin runs again and credentials are checked normally. If the
+credential verification is successful, then the Session plugin creates a new
+session to use for subsequent requests.
+
+When a new request comes in and a session is already present, then the Session
+plugin attaches the `ngx.ctx` variables to let the authentication
+plugin know that authentication has already occurred via session validation.
+As this configuration is a logical `OR` scenario, and forbidden anonymous
+access is desirable, you should configure the [Request Termination](/plugins/request-termination/) plugin on an anonymous consumer. 
+Failure to do so allows unauthorized requests. 
+For more information, see [multiple authentication](/gateway/authentication/#using-multiple-authentication-methods).
```

Comment:
```suggestion
As this configuration is a logical `OR` scenario and you likely want to forbid anonymous access, you should configure the [Request Termination](/plugins/request-termination/) plugin on an anonymous Consumer. 
If not configured, unauthorized requests will be allowed through. 
For more information, see [multiple authentication](/gateway/authentication/#using-multiple-authentication-methods) and learn how to [prevent unauthorized access in a multi-auth scenario](/how-to/authenticate-consumers-with-session-and-key-auth/).
```
I think it's useful to also link to the how-to guide on this, but you should keep the first link as well.